actors/util/adt/array.go

```diff
@@ -116,14 +117,14 @@ func (a *Array) write(root *amt.Root) error {
 func (a *Array) Get(k uint64, out runtime.CBORUnmarshaler) (bool, error) {
 	root, err := amt.LoadAMT(a.store.Context(), a.store, a.root)
 	if err != nil {
-		return false, errors.Wrapf(err, "array get failed to load root %v", a.root)
+		return false, xerrors.Errorf("array get failed to load root %v: %w", a.root, err)
```

Comment:
Maybe we need to talk about errors. We talked about it a few weeks ago but you keep (inadvertently?) making it inconsistent. What are you trying to achieve here that the prior state doesn't?

Reply:
Take a look at https://blog.golang.org/go1.13-errors

Pkg/errors is not canonical go, and the package is being deprecated: https://github.com/pkg/errors#roadmap

We can either use fmt.Errorf with the %w verb, or use xerrors.Errorf with the %w verb. We tend to prefer xerrors as it has the exact api, but includes stack information that is super helpful for debugging.

Reply:
Thanks for the links. I wish we could have had this information when we first talked about it before @frrist and I standardised this repo on pkg/errors!

FWIW I have only heard negative things from other Go devs about the new errors. We don't *have* to follow. But so long as we do something consistent so that new developers have a simple pattern to follow, I don't mind too much.